src/codemodder/codemods/base_codemod.py

```diff
@@ -22,6 +22,22 @@ class CodemodMetadata:
     DESCRIPTION: str  # TODO: this field should be optional
     NAME: str
     REVIEW_GUIDANCE: ReviewGuidance
+    REFERENCES: list = field(default_factory=list)
+
+    # TODO: remove post_init update_references once we add description for each url.
```

Comment:
This entire green block will be deleted once we add descriptions. I had already added the urls for each codemod so I decided to just automated and copy paste the url into the description field